packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -197,10 +197,11 @@ const helpers = {
               ...contentNegotiationResult,
             }));
         } else {
-          logger.trace(`Unable to find a content for ${mediaTypes}, returning an empty text/plain response.`);
+          logger.trace(`Unable to find a content for ${mediaTypes}, returning application/json+problem response.`);
+
           return right({
```

Comment:
This is probably making your test pass but this is providing the functionality that what we need. 

Assuming this spec is being used:

```
openapi: 3.0.2
paths:
  /todos:
    get:
      responses:
        200:
          description: Get Todo Items
          content:
            'text/plain':
              example: hello
```

This bit of code should be triggered when a matching status code is defined, but no schema or example can be found. 

Seeing as this does not have a 406 defined, Prism should not default with a 200 OK text/plain, by which I mean it should not be hitting this point in the code. This code is correct, but Prism should respond with a 406 Not Acceptable application/problem+json instead. 


This change fixes the symptom but not the cause.

Reply:
I agree, we should probably return a `left` saying "we cannot generate anything here" and then somewhere outside of this piece of code we'd be creating such response.